examples/2d/rect.rs

```diff
@@ -10,8 +10,13 @@ fn main() {
 fn setup(mut commands: Commands, mut materials: ResMut<Assets<ColorMaterial>>) {
     commands.spawn_bundle(OrthographicCameraBundle::new_2d());
     commands.spawn_bundle(SpriteBundle {
-        material: materials.add(Color::rgb(0.25, 0.25, 0.75).into()),
-        sprite: Sprite::new(Vec2::new(50., 50.)),
-        ..Default::default()
+        transform: Transform {
+            scale: Vec3::new(50.0, 50.0, 0.0),
+            ..Default::default()
+        },
```

Comment:
why do you set the scale here? is it to replace the `Sprite::new(Vec2::new(50., 50.))`?

Then you could do like https://github.com/bevyengine/bevy/blob/7356f1586d74039f840bcfcf24af3e21c23e3c18/examples/2d/contributors.rs#L100

Reply:
Ah, okay. That's what someone else suggested as well but neither of us were sure which one is the right way.
I tried something else now. Is it fine?